TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -204,14 +208,21 @@ codeElement:
 	onSizeErrorCondition |
 	notOnSizeErrorCondition
 
-//	 [TYPECOBOL]
+//	[TYPECOBOL]
 	| tcCodeElement;
 
 // what is here is not important as long as it is not epsilon
 tcCodeElement: PUBLIC | PRIVATE | UNSAFE;
 //	[/TYPECOBOL]
 
 
+//	[COBOL2002]
+// dummy definition of Cobol 2002 or V6 code elements, not used but required by Antlr.
+// see Cobol2002CodeElements.g4 for the proper definition.
+cobol2002Statement: UNSAFE;
```

Comment:
Ok dummy production has been created in order to not have many changes to be handled in CodeElementBuilder.
But at the first glance, this grammar in fact ambigüe even if theses are error cases. because UNSAFE is used as first token in both rules: tcCodeElement and cobol2002Statement, I think in this situation the first rule will match. 
So I think If this is the choice it's better than to use a distinct dummy token for each grammar extension like:
tcCodeElement: TC_EXTENSIONS;
cobol2002Statement: CBL2002_EXTENSIONS;

And also a token which will never be returned by the Scanner, because UNSAFE can be returned.

Reply:
Ok to include specific dummy tokens in next change (ALLOCATE support) but not directly in this PR as it is easier to keep the TokenList as it is.